dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/secret/ServiceIntegrationResourceTest.java

```diff
@@ -0,0 +1,170 @@
+import javax.servlet.http.HttpServletResponse;
+import javax.ws.rs.core.Response;
+import org.glassfish.jersey.media.multipart.ContentDisposition;
+import org.glassfish.jersey.media.multipart.FormDataBodyPart;
+import org.glassfish.jersey.media.multipart.FormDataMultiPart;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ServiceIntegrationResourceTest extends IntegrationTestBase {
+
+    static final String ADMIN_DEFAULT_ID = "dotcms.org.1";
+    static final String ADMIN_DEFAULT_MAIL = "admin@dotcms.com";
+    static final String ADMIN_NAME = "User Admin";
+
+    private static ServiceIntegrationResource serviceIntegrationResource;
+
+    private static final ObjectMapper ymlMapper = new ObjectMapper(new YAMLFactory())
+            //.enable(SerializationFeature.INDENT_OUTPUT)
+            .setVisibility(PropertyAccessor.FIELD, Visibility.ANY)
+            .findAndRegisterModules()
+            ;
+
+    @BeforeClass
+    public static void prepare() throws Exception {
+        //Setting web app environment
+        IntegrationTestInitService.getInstance().init();
+
+        final User user = mock(User.class);
+        when(user.getUserId()).thenReturn(ADMIN_DEFAULT_ID);
+        when(user.getEmailAddress()).thenReturn(ADMIN_DEFAULT_MAIL);
+        when(user.getFullName()).thenReturn(ADMIN_NAME);
+        when(user.getLocale()).thenReturn(Locale.getDefault());
+        when(user.isBackendUser()).thenReturn(true);
+
+        final WebResource webResource = mock(WebResource.class);
+        final InitDataObject dataObject = mock(InitDataObject.class);
+        when(dataObject.getUser()).thenReturn(user);
+
+        when(webResource.init(any(WebResource.InitBuilder.class))).thenReturn(dataObject);
+
+        //ServiceIntegrationAPI serviceIntegrationAPI = new ServiceIntegrationAPIImpl();
+
+        ServiceIntegrationHelper serviceIntegrationHelper = new ServiceIntegrationHelper();
+
+        serviceIntegrationResource = new ServiceIntegrationResource(webResource, serviceIntegrationHelper);
+    }
+
+    private InputStream createServiceDescriptorFile(final String fileName, final String serviceKey,
+            String serviceName, final String description,
+            final Map<String, Param> paramMap) throws IOException {
+        final ServiceDescriptor serviceDescriptor = new ServiceDescriptor(serviceKey, serviceName, description, "/black.png");
+
+        for (final Entry<String, Param> entry : paramMap.entrySet()) {
+            final Param param = entry.getValue();
+            serviceDescriptor.addParam(entry.getKey(),param.getValue(), param.isHidden(), param.getType(), param.getLabel(), param.getHint());
+        }
+
+        String basePath = System.getProperty("java.io.tmpdir");
+        basePath = Paths.get(basePath).normalize().toString();
+        final File file = new File(basePath,fileName);
+        ymlMapper.writeValue(file, serviceDescriptor);
+        System.out.println(file);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [System.out.println is used](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4623512)